src/components/posts/view/postsView.js

```diff
@@ -345,76 +331,68 @@ class PostsView extends Component {
-            defaultText={filterOptions[selectedOptionIndex]}
-            rightIconName="view-module"
-            rightIconType="MaterialIcons"
-            onDropdownSelect={this._handleOnDropdownSelect}
-            onRightIconPress={handleImagesHide}
-          />
-        )}
-
-        <FlatList
-          data={posts}
-          showsVerticalScrollIndicator={false}
-          renderItem={({ item }) =>
-            get(item, 'author', null) && (
-              <PostCard isRefresh={refreshing} content={item} isHideImage={isHideImage} />
-            )
-          }
-          keyExtractor={(content, i) => `${get(content, 'permlink', '')}${i.toString()}`}
-          onEndReached={() => this._loadPosts()}
-          removeClippedSubviews
-          refreshing={refreshing}
-          onRefresh={() => this._handleOnRefreshPosts()}
-          onEndThreshold={0}
-          initialNumToRender={10}
-          ListFooterComponent={this._renderFooter}
-          onScrollEndDrag={this._handleOnScroll}
-          ListEmptyComponent={this._renderEmptyContent}
-          refreshControl={
-            <ThemeContainer>
-              {({ isDarkTheme }) => (
-                <RefreshControl
-                  refreshing={refreshing}
-                  onRefresh={this._handleOnRefreshPosts}
-                  progressBackgroundColor="#357CE6"
-                  tintColor={!isDarkTheme ? '#357ce6' : '#96c0ff'}
-                  titleColor="#fff"
-                  colors={['#fff']}
-                />
-              )}
-            </ThemeContainer>
-          }
-          ref={ref => {
-            this.flatList = ref;
-          }}
+  return (
+    <View style={styles.container}>
+      {filterOptions && isShowFilterBar && (
+        <FilterBar
+          dropdownIconName="arrow-drop-down"
+          options={filterOptions}
+          selectedOptionIndex={selectedOptionIndex}
+          defaultText={filterOptions[selectedOptionIndex]}
+          rightIconName="view-module"
+          rightIconType="MaterialIcons"
+          onDropdownSelect={_handleOnDropdownSelect}
+          onRightIconPress={handleImagesHide}
+          customOption={customOption}
         />
-      </View>
-    );
-  }
-}
+      )}
+
+      <FlatList
+        data={posts}
+        showsVerticalScrollIndicator={false}
+        renderItem={({ item }) =>
+          get(item, 'author', null) && (
+            <PostCard isRefresh={refreshing} content={item} isHideImage={isHideImage} />
+          )
+        }
+        keyExtractor={(content, i) => `${get(content, 'permlink', '')}${i.toString()}`}
+        onEndReached={() => _loadPosts()}
+        removeClippedSubviews
+        refreshing={refreshing}
+        onRefresh={_handleOnRefreshPosts}
+        onEndThreshold={0}
+        initialNumToRender={10}
+        ListFooterComponent={_renderFooter}
+        onScrollEndDrag={_handleOnScroll}
+        ListEmptyComponent={_renderEmptyContent}
+        refreshControl={
+          <ThemeContainer>
```

Comment:
dont use theme container inside of refresh control, android has a bug about it. you can use outside of flatlist